src/barkhaPJun20/Bank.java

```diff
@@ -0,0 +1,88 @@
+package barkhaPJun20;
+
+public class Bank {
+	int debitCount=0;
+	int creditCount=0;
+	int balanceCount=0;
+	
+	static int overallDebitCount=0;
+	static int overallCreditCount=0;
+	static int overallBalanceCount=0;
+	
+	int debitAmnt;
+	int creditAmnt;
+	int currentBalance;
+	
+	
+	int user;
+	
+	void setInitialAmnt(int uID,int Amnt){
+		user=uID;
+		currentBalance=Amnt;
+	}
+	
+	void debitAmount( int dAmnt){
+		debitAmnt=dAmnt;
+		currentBalance=currentBalance-debitAmnt;
+		debitCount++;
+		overallDebitCount++;
+	}	
+	void creditAmount( int cAmnt){
+		creditAmnt=cAmnt;
+		currentBalance=currentBalance+creditAmnt;
```

Comment:
Same as above. creditAmnt is not required.

Reply:
Done